sessionist.tmux

```diff
@@ -14,6 +14,9 @@ tmux_option_new="@sessionist-new"
 default_key_bindings_promote_pane="@"
 tmux_option_promote_pane="@sessionist-promote-pane"
 
+default_key_bindings_join_pane="*"
```

Comment:
Why not `j` or `J`?

Reply:
in a <sup>(hyphenated-)</sup>word: self-interest.  [tmux-pain-control has those](https://github.com/tmux-plugins/tmux-pain-control#bindings) and i use it too :sweat_smile: